### PR TITLE
[FLINK-24777][docs]fix Processed (persisted) in-flight data descripti…

### PR DESCRIPTION
**Pull request by camilesing**
What is the purpose of the change
fix Processed (persisted) in-flight data description miss on Monitoring Checkpointing page

Brief change log
fix Processed (persisted) in-flight data description miss on Monitoring Checkpointing page

Verifying this change
Does this pull request potentially affect one of the following parts:
Dependencies (does it add or upgrade a dependency): (yes /**no**)
The public API, i.e., is any changed class annotated with @Public(Evolving): (yes / **no**)
The serializers: (yes / **no** / don't know)
The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
The S3 file system connector: (yes / **no** / don't know)
Documentation
Does this pull request introduce a new feature? (yes / **no**)
If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by camilesing** [comment]
@Myasuka Could you review this? thx

**Comment by Myasuka** [comment]
@camilesing I have assigned the ticket to you, and please get the assignment first before creating a PR for the next time.
The CI is still broken, please ensure the CI works fine first. If you think the broken CI has no relationship with your changes, please re-trigger the CI.
Last but not least, please fill the PR template instead of just paste the JIRA ticket link in the PR description.

**Comment by camilesing** [comment]
@flinkbot run azure

File: docs/content/docs/ops/monitoring/checkpoint_monitoring.md [hunk 1]
```diff
@@ -65,8 +65,7 @@ Note that for failed checkpoints, metrics are updated on a best efforts basis an
...
 - **Checkpointed Data Size**: The checkpointed data size over all acknowledged subtasks. If incremental checkpointing is enabled this value is the checkpointed data size delta.
-- **Processed in-flight data**: The approximate number of bytes processed during the alignment (time between receiving the first and the last checkpoint barrier) over all acknowledged subtasks.
-- **Persisted in-flight data**: The number of bytes persisted during the alignment (time between receiving the first and the last checkpoint barrier) over all acknowledged subtasks. This is > 0 only if the unaligned checkpoints are enabled.
+- **Processed (persisted) in-flight data**: The approximate number of bytes processed during the alignment (time between receiving the first and the last checkpoint barrier) over all acknowledged subtasks. `persisted in-flight data`in round brackets mean the number of bytes persisted during the alignment (time between receiving the first and the last checkpoint barrier) over all acknowledged subtasks. This is > 0 only if the unaligned checkpoints are enabled.
```
Comment (Myasuka) [inline docs/content/docs/ops/monitoring/checkpoint_monitoring.md:68]: I think statement below looks better:
`The approximate number of bytes processed (persisted) during the alignment (time between receiving the first and the last checkpoint barrier) over all acknowledged subtasks. Persisted data could be larger than zero only if the unaligned checkpoints are enabled.`

**Review by Myasuka — CHANGES_REQUESTED** [review]
Thanks for your contribution, please update the Chinese and English statements as suggested.

File: docs/content.zh/docs/ops/monitoring/checkpoint_monitoring.md [hunk 2]
```diff
@@ -74,8 +74,7 @@ Checkpoint 历史记录保存有关最近触发的 checkpoint 的统计信息，
 - **Latest Acknowledgement**：JobManager 接收到任何 subtask 的最新确认的时间（如果尚未收到确认，则不适用）。
 - **End to End Duration**：从触发时间戳到最后一次确认的持续时间（如果还没有收到确认，则不适用）。完整 checkpoint 的端到端持续时间由确认 checkpoint 的最后一个 subtask 确定。这个时间通常大于单个 subtask 实际 checkpoint state 所需的时间。
 - **Checkpointed Data Size**：所有已确认的 subtask 的 checkpoint 的数据大小。如果启用了增量 checkpoint，则此值为 checkpoint 数据的增量大小。
-- **Processed in-flight data**：在 checkpoint alignment 期间（从接收第一个和最后一个 checkpoint barrier 之间的时间）对所有已确认的 subtask 处理的大约字节数。
-- **Persisted in-flight data**：在 checkpoint alignment 期间（从接收第一个和最后一个 checkpoint barrier 之间的时间）对所有已确认的 subtask 持久化的字节数。仅当启用 unaligned checkpoint 时，此值大于 0。
+- **Processed (persisted) in-flight data**：在 checkpoint 对齐期间（从接收第一个和最后一个 checkpoint barrier 之间的时间）对所有已确认的 subtask 处理的大约字节数。如果启用了 unaligned checkpoint，此值会大于0。
```
Comment (Myasuka) [inline docs/content.zh/docs/ops/monitoring/checkpoint_monitoring.md:77]: I don't think the Chinese translation is correct. These statements certainlly missed the information of **persisted data**.

**Review by Myasuka — CHANGES_REQUESTED** [review]
Please update the Chinese translation part.

File: docs/content.zh/docs/ops/monitoring/checkpoint_monitoring.md (hunk 2 above)
Comment (camilesing) [reply docs/content.zh/docs/ops/monitoring/checkpoint_monitoring.md]: thx, I'm too careless

File: docs/content/docs/ops/monitoring/checkpoint_monitoring.md [hunk 3]
```diff
@@ -65,8 +65,7 @@ Note that for failed checkpoints, metrics are updated on a best efforts basis an
...
 - **Checkpointed Data Size**: The checkpointed data size over all acknowledged subtasks. If incremental checkpointing is enabled this value is the checkpointed data size delta.
-- **Processed in-flight data**: The approximate number of bytes processed during the alignment (time between receiving the first and the last checkpoint barrier) over all acknowledged subtasks.
-- **Persisted in-flight data**: The number of bytes persisted during the alignment (time between receiving the first and the last checkpoint barrier) over all acknowledged subtasks. This is > 0 only if the unaligned checkpoints are enabled.
+- **Processed (persisted) in-flight data**: The approximate number of bytes processed (persisted) during the alignment (time between receiving the first and the last checkpoint barrier) over all acknowledged subtasks. Persisted data could be larger than zero only if the unaligned checkpoints are enabled.
```
Comment (Myasuka) [inline docs/content/docs/ops/monitoring/checkpoint_monitoring.md:68]: I think a slash here could be better.
processed (persisted) during the alignment  --> processed/persisted during the alignment

File: docs/content.zh/docs/ops/monitoring/checkpoint_monitoring.md [hunk 4]
```diff
@@ -74,8 +74,7 @@ Checkpoint 历史记录保存有关最近触发的 checkpoint 的统计信息，
 - **Latest Acknowledgement**：JobManager 接收到任何 subtask 的最新确认的时间（如果尚未收到确认，则不适用）。
 - **End to End Duration**：从触发时间戳到最后一次确认的持续时间（如果还没有收到确认，则不适用）。完整 checkpoint 的端到端持续时间由确认 checkpoint 的最后一个 subtask 确定。这个时间通常大于单个 subtask 实际 checkpoint state 所需的时间。
 - **Checkpointed Data Size**：所有已确认的 subtask 的 checkpoint 的数据大小。如果启用了增量 checkpoint，则此值为 checkpoint 数据的增量大小。
-- **Processed in-flight data**：在 checkpoint alignment 期间（从接收第一个和最后一个 checkpoint barrier 之间的时间）对所有已确认的 subtask 处理的大约字节数。
-- **Persisted in-flight data**：在 checkpoint alignment 期间（从接收第一个和最后一个 checkpoint barrier 之间的时间）对所有已确认的 subtask 持久化的字节数。仅当启用 unaligned checkpoint 时，此值大于 0。
+- **Processed (persisted) in-flight data**：在 checkpoint 对齐期间（从接收第一个和最后一个 checkpoint barrier 之间的时间）对所有已确认的 subtask 处理（持久化）的大约字节数。如果启用了 unaligned checkpoint，持久化的字节数会大于0。
```
Comment (Myasuka) [inline docs/content.zh/docs/ops/monitoring/checkpoint_monitoring.md:77]: The translation is still not correct, the persisted number could be larger than zero only if enabled unalighed checkpoint. However, it does not mean we must get a positive number when we enabled unaligned checkpoint.

Comment (Myasuka) [reply docs/content.zh/docs/ops/monitoring/checkpoint_monitoring.md:77]: I also suggest using a slash here.

**Review by Myasuka — CHANGES_REQUESTED** [review]

File: docs/content/docs/ops/monitoring/checkpoint_monitoring.md (hunk 3 above)
Comment (camilesing) [reply docs/content/docs/ops/monitoring/checkpoint_monitoring.md:68]: Sounds good. but Flink UI shows processed (persisted):

i thought to keep them in agreement.

Comment (Myasuka) [reply docs/content/docs/ops/monitoring/checkpoint_monitoring.md:68]: A slash could mean the `or` semantics in English. However, brackets does not have such meaning in Chinese. I think a slash could be better here.

**Review by Myasuka — APPROVED** [review]
This PR LGTM, let's wait for the CI turned green.

**Pull request closed without merging**